data/README.md

```diff
@@ -1,3 +1,35 @@
+# Database Setup
+Follow the project readme to install Postgres
+
+Activate the venv environment to be sure you have python3 
+
+Install the following additional packages:
+pip install pandas
+pip install sqlalchemy
+pip install lxml
+pip install requests
+pip install xlrd
+
+
+## Run the Load Processes
+
+1. If you're not sure, set PYTHONPATH=.
+2. Ensure you are in the jobhopper project root directory (there should be a data\scripts\sql_loader.py underneath)
+3. Run python data\load_jobhopper_data.py
+
```

Comment:
Placeholder note for building out the readme in the future: 
* Add instructions for different OSes 
* requirements.txt should include any new required packages